Userland/Libraries/LibPDF/Parser.cpp

```diff
@@ -214,6 +214,7 @@ PDFErrorOr<NonnullRefPtr<NameObject>> Parser::parse_name()
             break;
 
         if (m_reader.matches('#')) {
+            m_reader.consume();
```

Comment:
```suggestion
        if (m_reader.consume('#')) {
```

Reply:
I don't think this is right. `consume('#')` will always consume a character, even if it doesn't end up matching `#`, which means that the name would be missing that character.

Reply:
Oh, that's a weird API lol. nvm then